test/models/ErrorsSpec.scala

```diff
@@ -59,21 +59,21 @@ class DesErrorsSpec extends SpecBase {
       Json.toJson(desMultiErrorModel) shouldBe desMultiErrorJson
     }
 
-    "Deserialize to a DesMultiError as expected" in {
-      desMultiErrorJson.as[DesMultiError] shouldBe desMultiErrorModel
+    "Deserialize to a MultiError as expected" in {
+      desMultiErrorJson.as[MultiError] shouldBe desMultiErrorModel
     }
 
   }
 
-  "The UnexpectedDesResponse object" should {
+  "The UnexpectedResponse object" should {
 
-    "Have the error code 'UNEXPECTED_DES_RESPONSE'" in {
-      UnexpectedDesResponse.code shouldBe "UNEXPECTED_DES_RESPONSE"
-    }
-
-    "Have the error reason 'The DES response did not match the expected format'" in {
-      UnexpectedDesResponse.reason shouldBe "The DES response did not match the expected format"
-    }
+//    "Have the error code 'UNEXPECTED_DES_RESPONSE'" in {
+//      UnexpectedResponse.code shouldBe "UNEXPECTED_DES_RESPONSE"
+//    }
+//
+//    "Have the error reason 'The DES response did not match the expected format'" in {
+//      UnexpectedResponse.reason shouldBe "The DES response did not match the expected format"
+//    }
```

Comment:
Do we need these?

Reply:
Yep, I'll add back in - and include more to cater for all the error messages